cabal.project

```diff
@@ -170,6 +171,7 @@ source-repository-package
     shelley/chain-and-ledger/dependencies/non-integer
     shelley/chain-and-ledger/executable-spec
     shelley/chain-and-ledger/shelley-spec-ledger-test
+    shelley-ma/impl
```

Comment:
Is this one needed already? I was only going to add it when we really use it (#2679).

Reply:
I only added it because the build failed.